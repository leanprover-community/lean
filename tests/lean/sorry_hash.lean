example : true :=
begin
sorry #, -- broken
end

example : true := by { sorry # } -- broken

example : true := begin { sorry # } end -- broken

example : true := by trivial
#che