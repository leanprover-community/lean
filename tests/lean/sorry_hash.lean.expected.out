sorry_hash.lean:3:6: error: invalid expression
sorry_hash.lean:4:0: error: sync
sorry_hash.lean:3:0: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:6:29: error: invalid expression
sorry_hash.lean:8:0: error: sync
sorry_hash.lean:6:23: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:8:0: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:8:32: error: invalid expression
sorry_hash.lean:8:36: error: sync
sorry_hash.lean:10:0: error: sync
sorry_hash.lean:8:26: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:8:36: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:10:0: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:10:0: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:11:0: error: invalid expression
sorry_hash.lean:10:21: error: don't know how to synthesize placeholder
context:
⊢ Type ?
sorry_hash.lean:11:0: error: command expected
